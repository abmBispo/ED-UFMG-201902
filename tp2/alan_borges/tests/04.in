1000 3 1
700 a
900 b
500 c