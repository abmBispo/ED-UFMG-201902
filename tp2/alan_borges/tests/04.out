1 c 500
2 a 700
3 b 900
